generators/utilities/src/main/resources/csrc/emulator.cc

```diff
@@ -16,6 +20,8 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <getopt.h>
+// enable verilator multithreading
```

Comment:
This comment seems strange to me? Does including this header really enable multithreading?

Reply:
yes

Reply:
along with the other stuff i added

Reply:
I agree with @colinschmidt. This header is just adding the VPI, which one of the two(?) verilog<>C interfaces. By itself it's not doing anything related to multithreading, although it may be necessary for some things you call later.

Reply:
this header is required. my verilator build fails without it

Reply:
it is required indirectly because of multithreading.

Reply:
I understand that, but the comment is confusing. The inclusion of this header does not enable multithreading, it adds the VPI interface.

Reply:
i modified the comment for clarity